internal/client/bluechi.go

```diff
@@ -0,0 +1,74 @@
+package client
+
+import (
+	"strconv"
+	"strings"
+)
+
+const (
+	BlueChiControllerConfdDirectory string = "/etc/bluechi/controller.conf.d/"
+	BlueChiAgentConfdDirectory      string = "/etc/bluechi/agent.conf.d/"
+)
+
+type BlueChiControllerConfig struct {
+	AllowedNodeNames []string
+	ManagerPort      *int64
+	LogLevel         *string
+	LogTarget        *string
+	LogIsQuiet       *bool
+}
+
+func (cfg BlueChiControllerConfig) Serialize() string {
```

Comment:
Not sure if it's an overkill or not. But did you consider using https://github.com/go-ini/ini?

Reply:
I had a look a the library and it seems it does not support multi-line values as we need (yet). For the `AllowedNodeNames` we need to apply a line break for the values and with the `go-ini/ini` library, those are surrounded by triple quotes `"""`. There is an open PR for it (https://github.com/go-ini/ini/pull/331), but we'd need to wait for this.

Reply:
OK, thanks for the details